code/game/objects/items/stacks/rods.dm

```diff
@@ -15,6 +15,23 @@
 	attack_verb = list("hit", "bludgeoned", "whacked")
 	hitsound = 'sound/weapons/grenadelaunch.ogg'
 
+/obj/item/stack/rods/cyborg
+	name = "metal rod"
+	desc = "Some rods. Can be used for building, or something."

```

Comment:
basically all of these are just copypasted values, leave them undefined and let the parent handle it